### PR TITLE
SD JWT DotNet

### PR DESCRIPTION
**Pull request by thomas-tran**
I'd to propose the initial implementation of [IETF SD-JWT specification](https://datatracker.ietf.org/doc/draft-ietf-oauth-selective-disclosure-jwt/) written in Dot Net

### CODE REVIEW

**Comment by tkuhrt** [comment]
1. Could you please update to use the [latest template](https://github.com/openwallet-foundation/project-proposals/blob/main/proposal-template.md)?
2. Can you confirm whether you have completed the [project intake form](https://docs.google.com/forms/d/e/1FAIpQLSeO1bDGHUP-ZpCo1uynm94YOxZlek6RhCH7o3FnX1lZSXXfSQ/viewform?fbzx=4351560609072672295)?
3. The Wallet Framework .NET project is looking to create an SD-JWT .NET source base as well. It would be good to work with the maintainers of the project to ensure no duplication of effort.

**Comment by thomas-tran** [comment]
@tkuhrt 
1. Could you please update to use the [latest template](https://github.com/openwallet-foundation/project-proposals/blob/main/proposal-template.md)?
Done
2. Can you confirm whether you have completed the [project intake form](https://docs.google.com/forms/d/e/1FAIpQLSeO1bDGHUP-ZpCo1uynm94YOxZlek6RhCH7o3FnX1lZSXXfSQ/viewform?fbzx=4351560609072672295)?
Yes
3. The Wallet Framework .NET project is looking to create an SD-JWT .NET source base as well. It would be good to work with the maintainers of the project to ensure no duplication of effort.
Will check with the owner

File: projects/sd-jwt-dotnet.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+None
+
+# Project License
+
+Apache 2.0.
+
+# Source Control
+
+[https://github.com/thomas-tran/sd-jwt-dotnet](https://github.com/thomas-tran/sd-jwt-dotnet)
```
Comment (tkuhrt) [inline projects/sd-jwt-dotnet.md:31]: Before we will be able to transfer this repo to the OpenWallet Foundation, we will need to do one of the options found here: https://github.com/openwallet-foundation/project-proposals/#fixing-dco-on-existing-repositories

File: projects/sd-jwt-dotnet.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+Thomas Tran([Github](https://github.com/thomas-tran))
+
+# Proposed Project Governance
+
+None
+
+# Links to Documented Governance Practices
+
+https://docs.google.com/forms/u/2/d/e/1FAIpQLSeO1bDGHUP-ZpCo1uynm94YOxZlek6RhCH7o3FnX1lZSXXfSQ/formResponse?pli=1
```
Comment (tkuhrt) [inline projects/sd-jwt-dotnet.md:60]: Unfortunately, it looks like this points to a blank form for me. When you hear back from the Linux Foundation with a document that you agree with, you can update that here.

File: projects/sd-jwt-dotnet.md (hunk 1 above)
Comment (thomas-tran) [reply projects/sd-jwt-dotnet.md:31]: Done

Comment (tkuhrt) [reply projects/sd-jwt-dotnet.md:31]: It looks like only the last three have been signed off. Could you make sure to sign-off all previous commits too?

Comment (thomas-tran) [reply projects/sd-jwt-dotnet.md:31]: Done, please let me know if I missed anything else

File: projects/sd-jwt-dotnet.md (hunk 2 above)
Comment (thomas-tran) [reply projects/sd-jwt-dotnet.md:60]: I haven't got any email form the Linux Foundation yet, I will update it once I got it

**Comment by tkuhrt** [comment]
I have added this to the November 15 TAC call. Please join us if you can to discuss with the larger community.

**Comment by thomas-tran** [comment]
> I have added this to the November 15 TAC call. Please join us if you can to discuss with the larger community.

Do you have the link to join?

File: projects/sd-jwt-dotnet.md (hunk 2 above)
Comment (skounis) [inline projects/sd-jwt-dotnet.md:60]: Is the form submitted? Do we have access to the submission?

File: projects/sd-jwt-dotnet.md (hunk 1 above)
Comment (skounis) [inline projects/sd-jwt-dotnet.md:31]: I would suggest a bit more detailed readme that at least provides some steps on how to set up the project and use the library (snippets). Similarly to the examples that follows:
* https://github.com/openwallet-foundation-labs/wallet-framework-dotnet#quickstart-guide
* https://github.com/openwallet-foundation-labs/sd-jwt-kotlin#checking-out-the-implementation
* https://github.com/openwallet-foundation-labs/sd-jwt-python#setup

**Comment by skounis** [comment]
> > I have added this to the November 15 TAC call. Please join us if you can to discuss with the larger community.
> 
> Do you have the link to join?

@thomas-tran, connection details are available on the TOC meetings page below:
* https://tac.openwallet.foundation/meetings/

File: projects/sd-jwt-dotnet.md (hunk 1 above)
Comment (thomas-tran) [reply projects/sd-jwt-dotnet.md:31]: I am working on the documentation now. I will update shortly

File: projects/sd-jwt-dotnet.md (hunk 2 above)
Comment (thomas-tran) [reply projects/sd-jwt-dotnet.md:60]: I've completed the form submission, but unfortunately, I haven't received any response from the Linux Foundation. I'm unsure about the next steps and would appreciate guidance on what to do next.

Comment (skounis) [reply projects/sd-jwt-dotnet.md:60]: Hi @thomas-tran, the next step is to join the next TAC meeting to discuss this proposal and the comments we collected in this PR. The next call is scheduled for Nov 29th, 7:00am Pacific Time (GMT-08:00)

* https://tac.openwallet.foundation/meetings/
* https://discord.com/channels/1022962884864643214/1080192375621685288

File: projects/sd-jwt-dotnet.md (hunk 1 above)
Comment (thomas-tran) [reply projects/sd-jwt-dotnet.md:31]: It is done in the develop branch

**Comment by tkuhrt** [comment]
@thomas-tran : I have added the proposal to tomorrow's TAC meeting for discussion. It would be great if you could join us at 7:00am Pacific Time (GMT-08:00). Please see https://tac.openwallet.foundation/meetings/ for dial in information.

**Review by davidejalexander — COMMENTED** [review]
Surely we would be better thinking of this as layers and this project is supporting a different context and use case from other implementations. They key thing is to ensure we can plug in different implementations for different use cases and ensure they are interoperable upstream for those who integrate them

**Comment by tkuhrt** [comment]
Approved at the [November 29, 2023 meeting](https://tac.openwallet.foundation/meetings/2023/2023-11-29/)

**Pull request merged**